apps/extension/src/app/features/activity-list/activity-list-v2.tsx

```diff
@@ -0,0 +1,47 @@
+import { useCallback } from 'react';
+import { Outlet } from 'react-router';
+
+import {
+  type ActivityLinkClickHandler,
+  ActivityList,
+  type OnChainActivity,
+  makeActivityLink,
+} from '@leather.io/features';
+
+import { openInNewTab } from '@app/common/utils/open-in-new-tab';
+import { useAccountActivity } from '@app/query/activity/account-activity.query';
+import { useCurrentNetworkState } from '@app/store/networks/networks.hooks';
+
+export function ActivityListV2() {
```

Comment:
Naming wise, it'd be more forward thinking to name this `ActivityList` and the old one `ActivityListLegacy` than version numbers.

Reply:
I can change the naming from `V1` / `V2` to - `CurrentName` - `CurrentNameLegacy`. 

This revamp will take quite a while I image so I thought it safer to give versions